src/pages/FriendsList.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import users from '../data/user';
+import TopBar from '../components/TopBar';
+import SearchBox from '../components/SearchBox';
+function FriendsList() {
+  const [searchClick, setSearchClick] = useState(false);
+  const [search, setSearch] = useState('');
+  const onSearchButtonClicked = () => {
+    setSearchClick(!searchClick);
+  };
+  const handleSearchInputChange = (e) => {
+    setSearch(e.target.value);
+  };
+  const userSearchResult = users.filter((user) => {
+    return user.name.includes(search);
+  });
+  return (
+    <Container>
+      <TopBar current="friends" onSearchButtonClicked={onSearchButtonClicked} />
+      {searchClick ? (
+        <SearchBox
+          searchClick={searchClick}
+          setSearchClick={setSearchClick}
+          handleInputChange={handleSearchInputChange}
+        />
+      ) : (
+        ''
+      )}
```

Comment:
```suggestion
      {searchClick && (
        <SearchBox
          searchClick={searchClick}
          setSearchClick={setSearchClick}
          handleInputChange={handleSearchInputChange}
        />
      )}
```
보통 `true`인 경우에만 무언가 실행하고 싶다면 이런식으로 하는게 많더라구요

Reply:
저도 배워갑니다!!!